data-raw/healthy-places/access-to-green-space/access-greenspace.R

```diff
@@ -0,0 +1,30 @@
+# Load packages
+library(tidyverse)
+library(geographr)
+
+# Load lookup dataset
+lookup_ltla_raw <- lookup_ltla_ltla
+
+# Filter the dataset for Welsh local authorities - LTLA code and LTLA name
+raw_welsh_greenspace <- data.frame ()
+raw_welsh_greenspace <- lookup_ltla_raw |>
+  filter(startsWith(ltla21_code, "W0"))|>
+  select(ltla21_code, ltla21_name)
+print (raw_welsh_greenspace)
+
+# Add new columns for the green space data
```

Comment:
Could you please add a couple of lines of comment with the url where you got the data from (e.g. `#source: www...... `) and a comment per column explaining briefly what each column denotes. The full explanation will eventually be added to the dataframe's documentation, but we need a short one here.